index.js

```diff
@@ -1547,7 +1552,7 @@
   //. Left('Cannot divide by zero')
   //. ```
   function Left(x) {
-    return new Either(sentinel, 'Left', x);
+    return new _Either('Left', x);
```

Comment:
Good point. We may need to tweak things in #232 in order to accommodate the current FL spec.